metricbeat/docs/modules/beat.asciidoc

```diff
@@ -20,6 +20,9 @@ UI in {kib}. To enable this usage, set `xpack.enabled: true` and remove any `met
 from the module's configuration. Alternatively, run `metricbeat modules disable beat` and
 `metricbeat modules enable beat-xpack`.
 
+NOTE: This module sends metrics to the monitoring index instead of the default
+index typically used by {metricbeat}. For more details about the monitoring
+index, see {ref}/config-monitoring-indices.html.
```

Comment:
Maybe add link text for all of these? Or is there a use-case for omitting it? This is how it looks right now:

<img width="752" alt="Screen Shot 2021-02-09 at 7 09 06 AM" src="https://user-images.githubusercontent.com/5618806/107383291-b7c19f80-6aa5-11eb-938f-fa33833ef3cf.png">

Reply:
That was a mistake. I will fix it. :-)